build.gradle.kts

```diff
@@ -12,6 +12,8 @@ plugins {
   scala
 }
 
+project.extensions.add("intellijVersions", IntellijVersions(project.properties["overrideBuildTarget"] as String?))
```

Comment:
In fact, to make the extension _pure_ (in the sense of _only being dependent on its input params, not global state), I'd pass the `Properties` object as a param to constructor, alongside the `overrideBuildTarget` project property